tasks/custom_task/custom_task.py

```diff
@@ -32,6 +33,7 @@ def log_invoke_task(name: str, module: str, task_datetime: str, duration: float,
     task_info = {
         "name": name,
         "module": module,
+        "pre_commit": os.environ.get('PRE_COMMIT', 0) == "1",
```

Comment:
```suggestion
        "pre_commit": os.environ.get('PRE_COMMIT', 0) == "0",
```

Reply:
`1` is correct in this case, since you'll have 1 if running pre-commit, so you get True